tests/testthat/test-fit_SE.R

```diff
@@ -1,9 +1,40 @@
 test_that("fit_SE errors as expected", {
   se <- SummarizedExperiment::SummarizedExperiment()
-  expect_error(fit_SE(se))
+  expect_error(fit_SE(se = 1), 
+               "'se' failed: Must inherit from class 'SummarizedExperiment'")
+  expect_error(fit_SE(se = se, data_type = 1), 
+               "'data_type' failed: Must be of type 'string', not 'double'")
+  expect_error(fit_SE(se = se, data_type = "dummy"), 
+               "'data_type' failed: Must be element of set {'single-agent','combination'}, ", 
+               fixed = TRUE)
+  expect_error(fit_SE(se = se, data_type = "single-agent", nested_identifiers = 1), 
+               "'nested_identifiers' failed: Must be of type 'character' (or 'NULL'), not 'double'", 
+               fixed = TRUE)
+  expect_error(fit_SE(se, averaged_assay = 1), 
+               "'averaged_assay' failed: Must be of type 'string', not 'double'")
+  expect_error(fit_SE(se, averaged_assay = "dummy"), 
+               "'dummy' is not on of the available assays: ''")
```

Comment:
This one is not exactly readable because se has no assay :(
Maybe use
```
se <- <- readRDS(system.file("testdata", 
                                 "finalMAE_combo_2dose_nonoise2.RDS", package = "gDRtestData")
  )[[2]]
```

Reply:
I'll add the tests on non-empty experiment.